cimetrics/env.py

```diff
@@ -170,8 +171,10 @@ def branch(self) -> Optional[str]:
 
     @property
     def pull_request_id(self) -> str:
-        assert self.is_pr
-        return os.environ["SYSTEM_PULLREQUEST_PULLREQUESTNUMBER"]
+        if self.is_pr:
+            return os.environ["SYSTEM_PULLREQUEST_PULLREQUESTNUMBER"]
+        else:
```

Comment:
Dubious, we probably want monitoring to happen only for a single, defined in config, branch.

Reply:
A more general implementation is probably to allow the config to specify an arbitrary set of branches, along with corresponding issues. That way release branches and so on can be supported.